components/pre_trial_pricing_modal.tsx/content.tsx

```diff
@@ -0,0 +1,296 @@
+            }}
+        >
+            {props.firstSvg}
+            {props.text}
+            {props.secondSvg}
+        </div>
+    );
+}
+
+function Card(props: CardProps) {
+    return (
+        <div className='PlanCard'>
+            {props.planLabel && props.planLabel}
+            <div
+                className='top'
+                style={{backgroundColor: props.topColor}}
+            />
+            <div className='bottom'>
+                <div className='plan_price_rate_section'>
+                    <h4>{props.plan}</h4>
+                    <h1 className={props.plan === 'Enterprise' ? 'enterprise_price' : ''}>{props.price}</h1>
+                    <p>{props.rate}</p>
+                </div>
+                <div className='plan_briefing'>
+                    <h4>{props.briefing.title}</h4>
+                    {props.briefing.items.map((i) => {
+                        return (
+                            <div
+                                className='item'
+                                key={i}
+                            >
+                                <i className='fa fa-circle bullet'/><p>{i}</p>
+                            </div>
+                        );
+                    })}
+                    {props.planDisclaimer && (
+                        <p className='disclaimer'><i className='icon-alert-outline'/>{props.planDisclaimer}</p>
+                    )}
+                </div>
+                <div>
+                    <button
+                        className={'plan_action_btn ' + props.buttonDetails.customClass}
+                        disabled={props.buttonDetails.disabled}
+                        onClick={props.buttonDetails.action}
+                    >{props.buttonDetails.text}</button>
+                </div>
+
+                {props.extraAction && (
+                    <button
+                        className='plan_action_btn'
+                        onClick={props.extraAction?.action}
+                    >{props.extraAction?.text}</button>
+                )}
+            </div>
+        </div>
+    );
+}
+
+function Content() {
+    const {formatMessage} = useIntl();
+
+    const contactSalesLink = useSelector((state: GlobalState) => getCloudContactUsLink(state, InquiryType.Sales));
+
+    const subscription = useSelector((state: GlobalState) => state.entities.cloud.subscription);
+    const product = useSelector((state: GlobalState) => {
+        if (state.entities.cloud.products && subscription) {
+            return state.entities.cloud.products[subscription?.product_id];
+        }
+        return undefined;
+    });
+
+    let isStarter = false;
+    if (product?.sku === CloudProducts.STARTER) {
+        isStarter = true;
+    }
+    return (
+        <div className='Content'>
+            <div className='self-hosted-alert'>
+                <h1>{formatMessage({id: 'pretrial_pricing_modal.lookingToSelfHost', defaultMessage: 'Looking to self-host?'})}</h1>
```

Comment:
Same for all `preTrial` references in this file

```suggestion
                <h1>{formatMessage({id: 'pricing_modal.lookingToSelfHost', defaultMessage: 'Looking to self-host?'})}</h1>
```